src/Spatial.Tests/Euclidean/LineSegment2DTests.cs

```diff
@@ -128,7 +128,7 @@ public void ClosestPointToWithinSegment(string start, string end, string point,
 
         [TestCase("0,0", "2,2", "1,0", "1,2", "1,1")]
         [TestCase("0,0", "2,2", "0,1", "2,1", "1,1")]
-        [TestCase("0,0", "2,2", "-1,-5", "-1,0", "-1,-1")]
+        [TestCase("-1,-1", "2,2", "-1,-5", "-1,0", "-1,-1")]
```

Comment:
We should add a test for parallel segments but that does not have to be in this PR.